sdk/evaluation/azure-ai-evaluation/tests/conftest.py

```diff
@@ -177,6 +177,34 @@ async def combined_call(*args, **kwargs):
     AsyncioRequestsTransport.send = original_transport_func
 
 
+@pytest.fixture
+def simple_conversation():
+    return {
+        "messages" : [
+            {
+                "content": "What is the capital of France?",
+                "role": "user",
+                "context": "Customer wants to know the capital of France"
```

Comment:
Are going to use this contract for image support later on? If so, does it mean we will reuse "context" for image?

Reply:
Context is currently string-only. If that changes, we'll address it then.